enter-container

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+podman run --privileged -it -v ./:/opt foreman-packaging
```

Comment:
I always use `--rm` as well. Also, is it wise to use `/opt` globally? Wouldn't `/opt/foreman-packaging` be better? And with is `--privileged` needed?